### DISCUSSION TITLE
Add missing Highlights channel for mentions and mention notification scroll to mention on click.

### DISCUSSION THREAD

**Discussion by TheFantasticLoki** [ppy/osu#17078] [Ideas] [answered]
I got mentioned by a user while afk and when I came back I clicked the notification saying I was mentioned. It didn't take me to the message in chat nor did the notification contain the contents of the message. There also is no highlights channel in lazer atm so I had no way of reading what he said as I could not find his message in chat anywhere.

I'd like to see at least a highlights channel implemented to solve this issue if not both. Also before someone comments I did MULTIPLE searches through discussions and issues with the keywords Highlight, Channel and mention and no existing threads showed for me.

**Comment by bdach** [comment]
I don't know about a "highlights channel" but the notification should probably scroll to the message. And maybe contain the message.

**Reply by TheFantasticLoki** [reply]
it didn't as stated above.

**Reply by bdach** [reply]
for the record you don't need to reply to every single message, i can read and read that in the original post, i'm just stating my agreement with it

**Comment by frenzibyte** [comment, marked as answer]
#17088
